DSSExcelPlugin/ExcelReader.cs

```diff
@@ -605,21 +605,17 @@ public DateTime GetDateFromCell(IRange range)
 
         private static bool IsRegular(List<DateTime> times)
         {
+            if (times.Count <= 1) { return false; }
             var temp = times;
-            var td = temp[1] - temp[0];
-            for (int i = 0; i < temp.Count; i++)
+            var td = temp[1] - temp[0]; // Get time difference.
+
+            // Check if time difference is the same throughout list
+            for (int i = 1; i < temp.Count - 1; i++)
```

Comment:
could this loop start at i=0?

Reply:
The reason it starts at i=1 is because we get the time difference from the first two values and then check if the time difference is maintained throughout the rest of the values. Since we get the time difference from subtraction times[0] from times[1], there wouldn't be a reason to start from 0 again, even though you technically could.